Source/WebCore/crypto/algorithms/CryptoAlgorithmAESKW.h

```diff
@@ -36,6 +36,7 @@ class CryptoAlgorithmAESKW final : public CryptoAlgorithm {
     static constexpr ASCIILiteral s_name = "AES-KW"_s;
     static constexpr CryptoAlgorithmIdentifier s_identifier = CryptoAlgorithmIdentifier::AES_KW;
     static Ref<CryptoAlgorithm> create();
+    static constexpr size_t s_extraSize = 64 / 8; // https://datatracker.ietf.org/doc/html/rfc3394#section-2.2.1
```

Comment:
nit: I think this can be protected instead of public because it's only used in a subclass and doesn't need to be available everywhere.

Reply:
using it in a static method in a file where I need the public visibility.